test/html/autocomplete.html

```diff
@@ -144,18 +158,21 @@ <h5>Pre-populate chips</h5>
         };
 
         const bigData = {};
+        const bigDataKeys = [];
         for (let i = 200; i >= 0; i--) {
             const randString = 'a' + Math.random().toString(36).substring(2);
             bigData[randString] = null;
+            bigDataKeys.push(randString)
         }
+        const fuse = new Fuse(bigDataKeys)
 
         const a1 = {
             data: defaultData
         };
 
         const a2 = {
             data: defaultData,
-            filterFunction: function (key_string, filter_string) {
+            compareFunction: function (key_string, filter_string) {
                 return true;
```

Comment:
I think returning true for a `compareFunction` is a bit weird. But it made sense for a thing called `filterFunction`. Don't you think?
Why did this line change?

Reply:
To my knowledge, `filterFunction` never existed, this is the only reference to it I can find. I'd consider using the name over `compareFunction` as I agree that it makes a bit more sense. Filter function to me, however does imply that things would be removed from the results but that's not the case. Maybe @samschurter could give some insight as they introduced this line (infact the whole file).